features/clip-path.yml

```
@@ -0,0 +1,19 @@
+name: clip-path
+description: The `clip-path` CSS property and SVG attribute sets an area of an element that should be displayed. Everything outside the area will be hidden.
```

For parallelism with the second sentence (and avoid "display" as a verb):

```suggestion
description: The `clip-path` CSS property and SVG attribute sets the visible area of an element. Everything outside the area will be hidden.
```